public/emails/ng_alert_notification.html

```diff
@@ -210,6 +210,21 @@
 {{Subject .Subject "{{.Title}}"}}
 
 {{ define "alert" }}
+
+  {{if ne .ImageURL "" }}
+  <tr style="vertical-align: top; padding: 0;" align="left">
+    <td colspan="2" style="word-break: break-word; -webkit-hyphens: auto; -moz-hyphens: auto; hyphens: auto; border-collapse: collapse !important; color: #222222; font-family: 'Open Sans', 'Helvetica Neue', 'Helvetica', Helvetica, Arial, sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none; margin: 0; padding: 24px 0 0;" align="left" valign="top">
+       <img src="{{.ImageURL}}" class="fluid-centered" alt="Alerting Panel" style="outline: none !important; text-decoration: none !important; -ms-interpolation-mode: bicubic; width: auto; clear: both; display: block; border: 0;" align="left" />
+		</td>
+	</tr>
+  {{end}}
+  {{if ne .EmbeddedImage "" }}
+  <tr style="vertical-align: top; padding: 0;" align="left">
+    <td colspan="2" style="word-break: break-word; -webkit-hyphens: auto; -moz-hyphens: auto; hyphens: auto; border-collapse: collapse !important; color: #222222; font-family: 'Open Sans', 'Helvetica Neue', 'Helvetica', Helvetica, Arial, sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none; margin: 0; padding: 24px 0 0;" align="left" valign="top">
+       <img src="cid:{{.EmbeddedImage}}" alt="Alerting Chart Attached Below" style="outline: none !important; text-decoration: none !important; -ms-interpolation-mode: bicubic; width: auto; clear: both; display: block; border: 0;" align="left" />
```

Comment:
Most webmail clients don't directly display `cid` image sources, hence the alt text "Alerting Chart Attached Below"

Reply:
TIL about `cid:` image links :D